go/cmd/dolt/cli/arg_parser_helpers.go

```diff
@@ -394,21 +385,6 @@ func CreateGlobalArgParser(name string) *argparser.ArgParser {
 var awsParams = []string{dbfactory.AWSRegionParam, dbfactory.AWSCredsTypeParam, dbfactory.AWSCredsFileParam, dbfactory.AWSCredsProfile}
 var ossParams = []string{dbfactory.OSSCredsFileParam, dbfactory.OSSCredsProfile}
 
-func ProcessBackupArgs(apr *argparser.ArgParseResults, scheme, backupUrl string) (map[string]string, error) {
-	params := map[string]string{}
-
-	var err error
-	switch scheme {
-	case dbfactory.AWSScheme:
-		err = AddAWSParams(backupUrl, apr, params)
-	case dbfactory.OSSScheme:
```

Comment:
I didn't see this `OSSScheme` mentioned in the documentation nor are they in the arg parser helper, so I just removed them and kept the AWSScheme code.